gevent/_sslgte279.py

```diff
@@ -575,6 +601,18 @@ def version(self):
             return None
         return self._sslobj.version()
 
+if PYPY or not hasattr(SSLSocket, 'timeout'):
+    # PyPy (and certain versions of CPython) doesn't have a direct
```

Comment:
Can you clarify? Why can't we depend on the `timeout` attribute defined in _socket2/_socket3?